user/tests/wiring/api/system.cpp

```diff
@@ -81,18 +81,21 @@ test(system_freememory) {
 }
 
 test(system_waitfor) {
+#if PLATFORM_ID!=10
     API_COMPILE(System.waitCondition([]{return WiFi.ready();}));
-
     API_COMPILE(waitFor(WiFi.ready, 10000));
     API_COMPILE(waitUntil(WiFi.ready));
+#endif
 }
 
 
 test(system_config_set) {
 
     API_COMPILE(System.set(SYSTEM_CONFIG_DEVICE_KEY, NULL, 123));
+#if PLATFORM_ID!=10
```

Comment:
This one really should be: `#if PLATFORM_ID >= 6 && PLATFORM_ID <= 8` or better yet `#if PLATFORM == photon`